From d74bb86a62f0b8a82fdbcf2fc5cc7ceca0278734 Mon Sep 17 00:00:00 2001
From: Robert Rose <robertroyrose@gmail.com>
Date: Sun, 18 Nov 2018 12:59:06 -0800
Subject: [PATCH 1/4] c-ares: support host build

Signed-off-by: Robert Rose <robertroyrose@gmail.com>
---
 package/c-ares/c-ares.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/c-ares/c-ares.mk b/package/c-ares/c-ares.mk
index 67254244a1..76e9687b37 100644
--- a/package/c-ares/c-ares.mk
+++ b/package/c-ares/c-ares.mk
@@ -15,3 +15,4 @@ C_ARES_LICENSE = MIT
 C_ARES_LICENSE_FILES = ares_mkquery.c
 
 $(eval $(autotools-package))
+$(eval $(host-autotools-package))
-- 
2.19.1

