From 87c615cb3191ee06fef6be1c39e88239b1523f5b Mon Sep 17 00:00:00 2001
From: Robert Rose <robertroyrose@google.com>
Date: Wed, 28 Nov 2018 10:02:32 -0800
Subject: [PATCH v3 1/2] c-ares: support host build

Signed-off-by: Robert Rose <robertroyrose@google.com>
---
 package/c-ares/c-ares.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/c-ares/c-ares.mk b/package/c-ares/c-ares.mk
index 67254244a1..76e9687b37 100644
--- a/package/c-ares/c-ares.mk
+++ b/package/c-ares/c-ares.mk
@@ -15,3 +15,4 @@ C_ARES_LICENSE = MIT
 C_ARES_LICENSE_FILES = ares_mkquery.c
 
 $(eval $(autotools-package))
+$(eval $(host-autotools-package))
-- 
2.20.0.rc0.387.gc7a69e6b6c-goog

